README.md

```diff
@@ -1,28 +1,54 @@
 # Pupil
-Open source eye tracking software platform that started as a thesis project at MIT. Pupil is a project in active, community driven development. Pupil mobile eye tracking hardware is accessible, hackable, and affordable. The software is open source and written in `Python` and `C++` when speed is an issue.
+<p align="center"><img src="https://via.placeholder.com/800x400?text=PLACEHOLDER"/></p>
+
+**Open source eye tracking platform.**
+
+Pupil is a project in active, community driven development. Pupil Core mobile eye tracking hardware is accessible, hackable, and affordable. The software is open source and written in `Python` and `C++` when speed is an issue.
+
 
 Our vision is to create tools for a diverse group of people interested in learning about eye tracking and conducting their eye tracking projects.
 
-Chat with us on [Discord](https://pupil-labs.com/chat "#pupil channel on DiscordApp").
+Chat with us on [Discord](https://pupil-labs.com/chat "Pupil Server on Discord").
 
-## Project Website
-For an intro to the Pupil mobile eye tracking platform have a look at the [Pupil Labs Website](http://pupil-labs.com "Pupil Labs").
+## Website
+For an intro to Pupil Core mobile eye tracking platform take a look at the [Pupil Labs Website](http://pupil-labs.com/products/core "Pupil Labs").
 
-## Getting Started
+## Documentation
+Read the user guide and developer docs at [docs.pupil-labs.com](https://docs.pupil-labs.com "Pupil Labs user guide and develper docs"). 
 
-<p align="center"><img src="https://via.placeholder.com/640x320?text=PLACEHOLDER"/></p>
+## Users
+[Download the latest apps](https://github.com/pupil-labs/pupil/releases/latest "Download Pupil Capture, Pupil Player, and Pupil Service application bundles")! You don't need to know how to write code to _use_ Pupil.
 
-## Building and Running Pupil from Source
+## Develelopers
+There are a number of ways you can extend the functionality of Pupil.
+- [Use the API](): Use the network based real-time API to communicate with Pupil over the network and integrate with your application. 
+- [Develop a Plugin](): Plugins are loaded at runtime from the app bundle. You don't need to setup dependencies and run from source unless you really need bare metal access.
+- [Run from Source](#installing-dependencies): Can't do what you need to do with the network based api or plugin? Then get ready to dive into the inner workings of Pupil and run from source!
 
 ### Installing Dependencies
+* [Ubuntu 18.04 LTS](./docs/dependencies-ubuntu18.md "Pupil dependency installation for Ubuntu 18.04") (recommended Linux distribution)
+* [Ubuntu 17.10 or lower](./docs/dependencies-ubuntu17.md "Pupil dependency installation for Ubuntu 17.10 or lower")
+* [macOS](./docs/dependencies-macos.md "Pupil dependency installation for macOS")
+* [Windows 10](./docs/dependencies-windows.md "Pupil dependency installation for Windows 10")
+
+### Running from source
```

Comment:
Does this section feel out of place here?

Reply:
This is now outdated... take a look at the latest version on my branch.
What do you mean by out of place?
That it should be at a different place in the README or that it should not be in the README at all?